code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -47,12 +48,16 @@
 	if(is_ctf_target(loc)) //don't reset from someone's hands.
 		return PROCESS_KILL
 	if(world.time > reset_cooldown)
-		forceMove(get_turf(src.reset))
-		for(var/mob/M in GLOB.player_list)
-			var/area/mob_area = get_area(M)
-			if(istype(mob_area, game_area))
-				to_chat(M, "<span class='userdanger'>\The [src] has been returned to base!</span>")
-		STOP_PROCESSING(SSobj, src)
+		reset_flag()
+
+/obj/item/ctf/proc/reset_flag()
+	forceMove(get_turf(src.reset))
+	for(var/mob/M in GLOB.player_list)
+		var/area/mob_area = get_area(M)
+		if(istype(mob_area, game_area))
+			to_chat(M, "<span class='userdanger'>\The [src] has been returned to base!</span>")
```

Comment:
```suggestion
			to_chat(M, "<span class='userdanger'>[src] has been returned to base!</span>")
```

The is added automatically.

Reply:
i think he wants it to be capitalized

Reply:
![image](https://user-images.githubusercontent.com/35135081/101687422-50449080-3a1f-11eb-8406-9896e18c50f0.png)

Yes.